### PR TITLE
MRD-30 - Automated a11y checks

### PR DESCRIPTION
**Pull request by jonnywyatt**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -249,6 +292,11 @@ workflows:
       - integration_test:
           requires:
             - build
+      - accessibility_test:
+          requires:
+            - build
+          context:
```
Comment (dazoakley) [inline .circleci/config.yml:298]: no need for the context unless you're going to report (back to slack) upon failure

File: .circleci/config.yml [hunk 2]
```diff
@@ -140,6 +144,45 @@ jobs:
...
+          name: Run wiremock
+          command: java -jar wiremock.jar --port 9091 --root-dir fake-manage-recalls-api/stubs
+          background: true
+      - run:
+          name: Run the node app.
+          command: npm run start:feature
+          background: true
+      - run:
+          name: Wait for node app to start
+          command: sleep 5
```
[the remark this answers is not in the document]
Comment (jonnywyatt) [reply .circleci/config.yml:175]: is there an example of this elsewhere?

File: .circleci/config.yml (hunk 1 above)
Comment (jonnywyatt) [reply .circleci/config.yml]: Removed

File: .circleci/config.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (jonnywyatt) [reply .circleci/config.yml]: Added

File: .circleci/config.yml [hunk 3]
```diff
@@ -140,6 +144,45 @@ jobs:
...
+      - run:
+          name: accessibility tests
+          command: npm run accessibility-test || true
+      - store_test_results:
+          path: test_results
```
[the remark this answers is not in the document]
Comment (dazoakley) [reply .circleci/config.yml:180]: ok, just worth checking it actually gets generated once the tests have run :)

**Review by dazoakley — DISMISSED** [review]
Noice 👌

**Review by dazoakley — APPROVED** [review]

**Pull request merged**